packages/material-ui-based-components/src/components/Slider.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback } from 'react';
+import { withRuleEngine } from '../shared/withRuleEngine';
+import { PROPS } from '../utils/types';
+import { Box, FormHelperText, InputLabel, Slider } from '@mui/material';
+
+const SliderComponent = (props: PROPS) => {
+    const { isError, name, label, required, description, enabled,
+        maximum, minimum, errorMessage, id, value, step }
+        = props;
+
+    function valuetext(value: number, label: any) {
```

Comment:
bind with usecallback hook

Reply:
rename the function name

Reply:
Binded with useCallback hook and changed the name to getValue.